src/control/server/main.go

```diff
@@ -130,6 +174,14 @@ func serverMain() error {
 		return err
 	}
 
+	if syscall.Getuid() == 0 {
+		log.Debugf("Dropping privileges...")
+		if err := drop(config.Username); err != nil {
+			log.Errorf("Failed to drop privileges: %s", err)
+			// TODO: don't continue as root
```

Comment:
I'm going to add an --insecure flag with the gRPC stuff so maybe we make it take a string of things that we'll allow to be insecure and make this an option to not drop privileges from root?

Reply:
can't we make that change when you add the insecure flag? seems a bit premature to add that capability in this patch